src/modules/mail/mail.service.ts

```diff
@@ -0,0 +1,39 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { MailerService } from '@nestjs-modules/mailer';
+import { ConfigService } from '@nestjs/config';
+
+import { ESubjectName } from './enums/subject-name.enum';
+import { EMailTemplate } from './enums/mail-template.enum';
+
+@Injectable()
+export class MailService {
+  constructor(
+    private readonly mailerService: MailerService,
+    private readonly configService: ConfigService,
+  ) {}
+  public async sendMail(
```

Comment:
please add the empty line above) even a book has paragraphs, its will be easier to read files